subprojects/core/src/main/java/org/gradle/groovy/scripts/internal/InitialPassStatementTransformer.java

```diff
@@ -82,7 +83,7 @@ public Statement transform(SourceUnit sourceUnit, Statement statement) {
                     );
                 }
 
-                return null;
+                return statement;
```

Comment:
⭕️ Consider transforming the `plugins { ... }` block call at this point to pass `pluginsBlockLineNumber` as its first argument, i.e., `plugins { s* } => plugins($pluginsBlockLineNumber) { s* }` and change `ProjectScript.plugins(Closure)` to `ProjectScript.plugins(int lineNumber, Closure closure)`.

Reply:
Great suggestion, thanks. da267ac0ed3d0c489c47c629ed3507c7ec064981